### PR TITLE
feat: makes external call to get phone, grower and funder

### LINKED ISSUE #5: Upload imported file to S3 before starting processing
Every file import should be uploaded to s3 before processing begins.

The timestamp should be prepended to the filename before being saved to s3.

S3 credentials will be provided as environment variables.

### PR DESCRIPTION
**Pull request by Kpoke**
fixes #5

### CODE REVIEW

**Comment by Kpoke** [comment]
The upload is currently done to s3 before processing begins. The s3 credentials are also being supplied as environment variables, although I changed the names to be clearer. And I prepended the timestamp to each file.

**Comment by Kpoke** [comment]
Also, added support for sorting by grower and funder, also made some refactors on the earning repository as well as changes to the names of the s3 environment variables

**Comment by Kpoke** [comment]
Converted to draft because merging it would break the API as the web-map API has not been updated

**Review by dadiorchen — APPROVED** [review]

File: server/handlers/earningsHandler.js [hunk 1]
```diff
@@ -91,15 +91,13 @@ const earningsBatchGet = async (req, res) => {
...
-      .on('error', (error) => {
-        console.log('error', error.message);
-        throw new HttpError(422, error.message);
-      })
-      .on('end', () => csvStream.end());
+    // using for await due to the async call that is made
+    for await (const row of earningsStream) {
+      const earningRow = await BatchEarning({ ...row });
+      csvStream.write(earningRow);
+    }
```
[the remark this answers is not in the document]
Comment (ZavenArra) [reply server/handlers/earningsHandler.js:98]: it's fine.

**Pull request merged**